daml-lf/spec/daml-lf-1.rst

```diff
@@ -2087,84 +2103,98 @@ Int64 functions
+  in ``Some``. If the input does not match the regexp ``[+-]?\d+`` or
   if the result of the conversion overflows, returns ``None``.
 
   [*Available since version 1.5*]
 
-Decimal functions
+Numeric functions
 ~~~~~~~~~~~~~~~~~
 
-* ``ADD_DECIMAL : 'Decimal' → 'Decimal' → 'Decimal'``
+* ``ADD_NUMERIC : ∀ (α : nat) . 'Numeric' α → 'Numeric' α  → 'Numeric' α``
 
-  Adds the two decimals. Throws an error in case of overflow.
+  Adds the two decimals.  The scale of the inputs and the ouput is
+  given by the type parameter `α`.  Throws an error in case of
+  overflow.
 
-* ``SUB_DECIMAL : 'Decimal' → 'Decimal' → 'Decimal'``
+* ``SUB_NUMERIC : ∀ (α : nat) . 'Numeric' α → 'Numeric' α → 'Numeric' α``
 
-  Subtracts the second decimal from the first one. Throws an error
-  if overflow.
+  Subtracts the second decimal from the first one.  The
+  scale of the inputs and the ouput is given by the type parameter
+  `α`.  Throws an error if overflow.
 
-* ``MUL_DECIMAL : 'Decimal' → 'Decimal' → 'Decimal'``
+* ``MUL_NUMERIC : ∀ (α : nat) . 'Numeric' α → 'Numeric' α → 'Numeric' α``
 
   Multiplies the two decimals and rounds the result to the closest
-  multiple of ``10⁻¹⁰`` using `banker's rounding convention
-  <https://en.wikipedia.org/wiki/Rounding#Round_half_to_even>`_.
-  Throws an error in case of overflow.
+  multiple of ``10⁻ᵅ`` using `banker's rounding convention
+  <https://en.wikipedia.org/wiki/Rounding#Round_half_to_even>`_.  The
+  scale of the inputs and the ouput is given by the type parameter
+  `α`. Throws an error in case of overflow.
 
-* ``DIV_DECIMAL : 'Decimal' → 'Decimal' → 'Decimal'``
+* ``DIV_NUMERIC : ∀ (α : nat) . 'Numeric' α → 'Numeric' α → 'Numeric' α``
 
   Divides the first decimal by the second one and rounds the result to
-  the closest multiple of ``10⁻¹⁰`` using `banker's rounding
-  convention
-  <https://en.wikipedia.org/wiki/Rounding#Round_half_to_even>`_. Throws
-  an error in case of overflow.
+  the closest multiple of ``10⁻ᵅ`` using `banker's rounding convention
+  <https://en.wikipedia.org/wiki/Rounding#Round_half_to_even>`_ (where
+  `n` is given as the type parameter).  The scale of the inputs and
+  the ouput is given by the type parameter `α`.  Throws an error in
+  case of overflow.
 
-* ``ROUND_DECIMAL : 'Int64' → 'Decimal' → 'Decimal'``
+* ``ROUND_NUMERIC : ∀ (α : nat) . 'Int64' → 'Numeric' α → 'Numeric' α``
 
-  Round the decimal to the closest multiple of ``10ⁱ`` where ``i`` is
-  integer argument.  Rounds the decimal argument to the closest
-  multiple of ``10ⁱ`` where ``i`` is integer argument. In case the
-  value to be rounded is exactly half-way between two multiples,
-  rounds toward the even one, following the `banker's rounding
-  convention
-  <https://en.wikipedia.org/wiki/Rounding#Round_half_to_even>`_. Throws
-  an exception if the integer is not between -27 and 10 inclusive.
+  Rounds the decimal to the closest multiple of ``10ⁱ`` where ``i`` is
```

Comment:
Is this actually true? Isn't it rounding to `i` digits after the dot?

Reply:
you'r right it is the the closet multiple of ``10⁻ⁱ`` !